### PR TITLE
fix: remove cartography part 9 SQPIT-772

### PR DESCRIPTION
**Pull request by billypchan**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQPIT-772" title="SQPIT-772" target="_blank">TaskSQPIT-772</a>  Remove Cartography
  </td></table>
  <br />
 

remove cartography from: 
- IconActionCell (tested covered in `ConversationOptionsViewControllerTests`)
- UserConnectionView (tested in `UserConnectionViewTests`)
- LocationPreviewController (tested in `LocationPreviewControllerTests`)
- CameraKeyboardPermissionsCell (tested in `CameraKeyboardViewControllerTests`)
- InputBar (ConversationInputBarViewControllerAudioRecorderSnapshotTests)
- AudioMessageView (AudioMessageViewTests)
- EmojiKeyboardViewController (covered in `CanvasViewControllerTests`)
- UserConnectionView (UserConnectionViewTests)
- LocationSendViewController (tested in `LocationSendViewControllerTests`)
- CanvasViewController (tested in `CanvasViewControllerTests`)
- CallQualityViewController (CallQualityControllerTests)
- LocationPreviewController (LocationPreviewControllerTests)
- DotView (tested in `AccountViewSnapshotTests`)
- SearchHeaderViewController (tested in `AddParticipantsViewControllerSnapshotTests`)
- LocationSelectionViewController
- AccentColorPickerController

Share extension:
- NotSignedInViewController
- SendingProgressViewController

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/CameraKeyboard/CameraKeyboardPermissionsCell.swift [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // Copyright (C) 2018 Wire Swiss GmbH
 //
 // This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
+// it under the terms of the GNU General License as published by
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/Conversation/InputBar/CameraKeyboard/CameraKeyboardPermissionsCell.swift:6]:
```suggestion
// it under the terms of the GNU General Public License as published by
```

File: Wire-iOS/Sources/UserInterface/Location/LocationSelectionViewController.swift [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
+//
 
 import WireDataModel
-import Cartography
 import MapKit
 import CoreLocation
 import UIKit
 
-protocol LocationSelectionViewControllerDelegate: AnyObject {
+protocol LocationSelectionViewControllerDelegate: class {
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/Location/LocationSelectionViewController.swift:24]:
```suggestion
protocol LocationSelectionViewControllerDelegate: AnyObject {
```

File: Wire-iOS/Sources/UserInterface/Location/LocationSendViewController.swift [hunk 3]
```diff
@@ -1,22 +1,21 @@
-// 
+//
 // Wire
 // Copyright (C) 2016 Wire Swiss GmbH
-// 
+//
 // This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
+// it under the terms of the GNU General License as published by
```
Comment (billypchan) [inline Wire-iOS/Sources/UserInterface/Location/LocationSendViewController.swift:6]:
```suggestion
// it under the terms of the GNU General Public License as published by
```

**Review by agisilaos — APPROVED** [review]

File: Wire-iOS/Sources/UserInterface/Location/LocationSendViewController.swift [hunk 4]
```diff
@@ -64,27 +63,32 @@ final class LocationSendViewController: UIViewController {
         [addressLabel, sendButton, separatorView].forEach(containerView.addSubview)
     }
 
-    fileprivate func createConstraints() {
-        constrain(view, containerView, separatorView, addressLabel, sendButton) { view, container, separator, label, button in
-            container.edges == inset(view.edges, 24, 0)
-            label.leading == container.leading
-            label.trailing <= button.leading - 12 ~ 1000.0
```
Comment (David-Henner) [inline Wire-iOS/Sources/UserInterface/Location/LocationSendViewController.swift:71]: priority has been ignored there too

File: Wire-iOS/Sources/UserInterface/Conversation/InputBar/InputBar.swift [hunk 5]
```diff
@@ -264,54 +263,61 @@ final class InputBar: UIView {
...
-            rightAccessoryView.width == 0 ~ 750.0
-            rightAccessoryView.bottom == buttonContainer.top
-
-            buttonContainer.top == textView.bottom
-            textView.top == textView.superview!.top
-            textView.leading == leftAccessoryView.trailing
-            textView.trailing <= textView.superview!.trailing - 16
-            textView.trailing == rightAccessoryView.leading
-            textView.height >= 56
-            textView.height <= 120 ~ 1000.0
```
[the remark this answers is not in the document]
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Conversation/InputBar/InputBar.swift:285]: priority = 1000 is default priority and no need to specify

File: Wire-iOS/Sources/UserInterface/Location/LocationSendViewController.swift (hunk 4 above)
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Location/LocationSendViewController.swift:71]: priority = 1000 is default priority and no need to specify

File: Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/FileTransfer/AudioMessageView.swift [hunk 6]
```diff
@@ -132,39 +129,44 @@ final class AudioMessageView: UIView, TransferView {
...
-            loadingView.center == selfView.center
-        }
-
+        [self,
+             playButton,
+             timeLabel,
+             downloadProgressView,
+             playerProgressView,
+             waveformProgressView,
+             loadingView].prepareForLayout()
```
[the remark this answers is not in the document]
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/FileTransfer/AudioMessageView.swift]: fixed

**Review by David-Henner — APPROVED** [review]

**Pull request merged**